docs/cookbook/displaying_content_with_simple_templates.md

```diff
@@ -63,11 +61,11 @@ In the example above you used the `ez_render_field` Twig function to render the
 </div>
 ```
 
-You can also make use of different other [Twig functions](../guide/content_rendering.md#twig-functions-reference), for example [ez\_field\_value](../guide/content_rendering.md#ez95field95value), which renders the value of the Field without a template.
+You can also make use of different other [Twig functions](../guide/content_rendering.md#twig-functions-reference), for example [`ez_field_value`](../guide/content_rendering.md#ez_field_value), which renders the value of the Field without a template.
```

Comment:
For: You can also make use of different other
Is other the name of Twig functions? If so please change to 'Other', if not it should be changed to 'different' or 'other'.